draft-ietf-gnap-core-protocol.md

```diff
@@ -3334,22 +3372,21 @@ MUST validate all components of the signed message to ensure that nothing
 has been tampered with or substituted in a way that would change the nature of
 the request.
 
+When a keep proofing mechanism is bound to an access token, the access token MUST be covered 
```

Comment:
keep -> key

Reply:
Also, I think this statement is aimed at us (the protocol developers) rather than implementers. IOW, I suggest to remove it.

Reply:
This statement is aimed any anyone who would write a key proofing method to extend the set here. Flexibility in this regard is foundational to the protocol. I think the sentiment needs to stay but it can be reworded or relocated if needed (though I'd suggest a separate PR/issue for this, perhaps under #134 that's collecting these items).